cardano-node/src/Cardano/Node/Tracing/Tracers/Consensus.hs

```diff
@@ -736,9 +736,24 @@ instance LogFormatting (FetchDecision [Point header]) where
              ]
 
 instance MetaTrace (FetchDecision [Point header]) where
+    namespaceFor (Left FetchDeclineChainIntersectionTooDeep)
```

Comment:
To make this fully functional namespaces needs more, so I propose to do a simple dispatch here on the runtime object